package.json

```diff
@@ -6,8 +6,10 @@
   "scripts": {
     "start": "node server",
     "env": "cp env.dist .env",
-    "db": "knex migrate:latest && knex seed:run",
-    "test": "rm dev.db && npm run db && lab -t 100 -c --verbose --colors --assert code --timeout 5000 && npm run lint",
+    "knex": "knex migrate:latest && knex seed:run",
+    "db:new": "createdb publish && npm run knex",
```

Comment:
You should remove the `createdb` and `dropdb` commands from here since a user can arbitrarily name their database now. What about an example in the README instead?